core-bundle/src/Routing/ResponseContext/ContaoWebpageResponseContext.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\ResponseContext;
+
+use Contao\PageModel;
+
+class ContaoWebpageResponseContext extends WebpageResponseContext
+{
+    public function __construct(PageModel $pageModel)
+    {
+        $this
+            ->setTitle($pageModel->pageTitle ?: $pageModel->title ?: '')
+            ->setMetaDescription(str_replace(["\n", "\r", '"'], [' ', '', ''], $pageModel->description ?: ''))
```

Comment:
what about meta robots here?

Reply:
Fixed in d829f75ae177c68c310bf3f52f495a853b39e26b.